vignettes/mmrm_review_methods.Rmd

```diff
@@ -246,8 +246,32 @@ MODEL FEV1 = / ddfm=satterthwaite solution chisq;
 ## Ante-dependence (homogeneous)
 
 ### mmrm
-<pre><code>mmrm::mmrm(formula = FEV1 ~ <b>ah(VISITN | USUBJID)</b>, 
+<pre><code>mmrm::mmrm(formula = FEV1 ~ <b>ad(VISITN | USUBJID)</b>, 
   data = fev_data, reml = FALSE)
 </code></pre>
 
-## {-}
+# Benchmarking
+```{r, cache=TRUE, warning=FALSE, message=FALSE}
```

Comment:
@danielinteractive What's the opinion on caching? Probably better to save the intermediate output into a table and just present it below?

Reply:
@BrianLang hm good question - maybe better to save explicitly manually results into a table or object and then save in the repo, since the caching stuff is difficult to make transparent in the repo.